src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
-<app-layout>
+<app-navbar>
```

Comment:
not keen on this implementation. Why navbar is wrapping router outlet? Should't be like this in your main app file?
<app-navbar>
<router-outlet></router-outlet>
<footer, or other stuff you need>

Reply:
Not sure about this one, I have pushed changes